src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/FacebookSignupModalComponentObject.java

```diff
@@ -17,18 +15,18 @@
 
   @FindBy(css = "button[name='__CONFIRM__']")
   private WebElement appTermsConfirmButton;
-  @FindBy(css = ".UserLoginFacebookLeft input[name='username']")
+  @FindBy(css = "#signupUsername")
   private WebElement usernameField;
-  @FindBy(css = ".UserLoginFacebookLeft input[name='password']")
+  @FindBy(css = "#signupPassword")
   private WebElement passwordField;
-  @FindBy(css = ".UserLoginFacebookLeft input[name='email']")
+  @FindBy(css = "#signupEmail")
   private WebElement emailField;
   @FindBy(css = ".UserLoginFacebookRight input[name='username']")
   private WebElement existingUsernameField;
   @FindBy(css = ".UserLoginFacebookRight input[name='password']")
   private WebElement existingPasswordField;
```

Comment:
`existingUsernameField` and `existingPasswordField` are never used it. Please delete it